lib/component-data/mutations.js

```diff
@@ -35,6 +35,10 @@ export default {
     _.set(state, 'ui.currentlySaving', isSaving);
     return state;
   },
+  [CURRENTLY_RESTORING]: (state, isSaving) => {
+    _.set(state, 'ui.currentlyRestoring', isSaving);
```

Comment:
What do you think if we rename `isSaving` in here to be `isRestoring`?
```suggestion
    _.set(state, 'ui.currentlyRestoring', isRestoring);
```

Reply:
Definitely should be isRestoring, I just cut and pasted the CURRENTLY_SAVING function above it and forgot to update param name.